src/Pages/SJSUDiscordBot/GoogleLogin.js

```diff
@@ -0,0 +1,128 @@
+import React, {useState} from 'react';
+import {GoogleLogin,
+  GoogleLogout} from 'react-google-login';
+import { GOOGLE_API_CLIENT_ID } from '../../config/config.json';
+import { getTempUser, addUser } from '../../APIFunctions/GoogleLoginPage.js';
+
+function Verify(){
+  const [isLogined, setLoginState] = useState(false);
+  const [accessToken, setAccessToken] = useState('');
+  const [username, setUsername] = useState('');
+  const [discordID, setDiscordID] = useState(0);
+  const [id, setID] = useState(window.location.href.split('/')[5]);
+  const [validID, setValidID] = useState(false);
+  const [loginFailed, setLoginFailed] = useState(false);
+  const [logoutFailed, setLogoutFailed] = useState(false);
+  const [SJSUEmail, setSJSUEmail] = useState(false);
+
+  const getUser = async () => {
+    await getTempUser(id).then(res => {
+      if(!res.error){
```

Comment:
```suggestion
      if (!res.error){
```
Minor complaint: it's better style to add a space between `if ()`